frontend/src/components/PasswordCreationDoubleInput.tsx

```diff
@@ -46,9 +47,11 @@ const usePasswordComplexity = (password: string): PasswordComplexity => {
         improvementsText: [],
       });
     } else {
-      estimatePasswordComplexity(deferredPassword, t).then((response) =>
-        setResult(response),
-      );
+      loadPromise
+        .then(({ estimatePasswordComplexity }) =>
+          estimatePasswordComplexity(deferredPassword, t),
+        )
+        .then((response) => setResult(response));
```

Comment:
will this potentially 'queue up' a bunch of `.then` tasks and then run them all at once when the import loads?

e.g. it might trigger multiple times if the user started typing a password before it finishes loading? Further, if the tasks run out of order, it might set the wrong score on the form?

Reply:
It will, but I wouldn't worry too much about it? 🤷